wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -350,49 +349,55 @@ p {
 		order: 2;
 	}
 
-	.lp-content .lp-details .lp-links,
+	.lp-content .lp-details .lp-actions,
 	.lp-suggestion {
 		display: block;
 	}
 }
 
-.lp-content .lp-details .lp-links {
+.lp-content .lp-details .lp-actions {
 	margin: 1.25rem 0;
 }
 
-.lp-content .lp-details .lp-links li {
+.lp-content .lp-details .lp-actions > li {
```

Comment:
Slight CSS performance improvement, so that we're not scanning every list item in the DOM